src/components/SideBar/SideBar.tsx

```diff
@@ -10,6 +10,7 @@ import ToolTipHover from '../ToolTipHover'
 import './sideBar.css'
 
 import { composeClasses } from 'lib/classes'
+import { Flex, Skeleton } from '..'
```

Comment:
This way, we could bring in code we don't want, let's do the default import of each one.